tests/pc/test_po_cleanup.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import logging
+from tests.common.utilities import wait_until
+from tests.common import config_reload
+
+
+pytestmark = [
+    pytest.mark.topology('any'),
+]
+
+@pytest.fixture(autouse=True)
+def ignore_expected_loganalyzer_exceptions(duthost, loganalyzer):
+    """
+        Ignore expected failures logs during test execution.
+
+        LAG tests are triggering following syncd complaints but the don't cause
+        harm to DUT.
+
+        Args:
+            duthost: DUT fixture
+            loganalyzer: Loganalyzer utility fixture
+    """
+    # when loganalyzer is disabled, the object could be None
+    if loganalyzer:
+        ignoreRegex = [
+            ".*ERR swss#orchagent: :- removeLag.*",
+        ]
+        loganalyzer.ignore_regex.extend(ignoreRegex)
+        expectRegex = [
+            ".*teamd#teammgrd: :- cleanTeamProcesses.*",
+            ".*teamd#teamsyncd: :- cleanTeamSync.*"
+        ]
+        loganalyzer.expect_regex.extend(expectRegex)
+
+    yield
```

Comment:
I don't think you need to add yield if no clean up is required

Reply:
@yxieca Done